_posts/2016-06-24-myeong-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Myeong: introduction"
+date:   2016-06-24
+author: Myeong Lee
+---
+<link href="../assets/css/myeong.css" rel="stylesheet"></link>
+
+# Myeong Lee (Ph.D. Student)
```

Comment:
You don't need a title. That gets automagically generated from the `title` field in the header!